### PR TITLE
data-source/alicloud_ecs_network_interfaces: add attribute ipv6_sets

### PR DESCRIPTION
**Pull request by shanye997**

### CODE REVIEW

**Comment by shanye997** [comment]
approved

File: alicloud/data_source_alicloud_ecs_network_interfaces.go [hunk 1]
```diff
@@ -417,6 +422,16 @@ func dataSourceAlicloudEcsNetworkInterfacesRead(d *schema.ResourceData, meta int
 		}
 		mapping["associated_public_ip"] = associatedPublicIps
 
+		if v := object["Ipv6Sets"].(map[string]interface{})["Ipv6Set"].([]interface{}); len(v) > 0 {
```
Comment (ChenHanZhang) [inline alicloud/data_source_alicloud_ecs_network_interfaces.go:425]: 这个Ipv6Set 会不会存在接口不返回的情况，会的话这里会出现panic

**Comment by shanye997** [comment]
approved

**Comment by shanye997** [comment]
approved

**Review by shanye997 — COMMENTED** [review]
approved

File: alicloud/data_source_alicloud_ecs_network_interfaces.go (hunk 1 above)
Comment (shanye997) [reply alicloud/data_source_alicloud_ecs_network_interfaces.go:438]: 已优化

**Review by shanye997 — COMMENTED** [review]
approved

**Review by shanye997 — COMMENTED** [review]
approved

**Review by shanye997 — COMMENTED** [review]
approved

**Pull request merged**